GVFS/GVFS.UnitTests/Virtualization/FileSystemCallbacksTests.cs

```diff
@@ -185,15 +159,6 @@ public void IsReadyForExternalAcquireLockRequests()
                 fileSystemVirtualizer: fileSystemVirtualizer))
             {
                 string denyMessage;
-                fileSystemCallbacks.IsReadyForExternalAcquireLockRequests(
-                    new NamedPipeMessages.LockData(
-                        pid: 0,
-                        isElevated: false,
-                        checkAvailabilityOnly: false,
-                        parsedCommand: "git dummy-command"),
-                    out denyMessage).ShouldBeFalse();
-                denyMessage.ShouldEqual("Waiting for mount to complete");

```

Comment:
Rather than removing this scenario from the test (trying to acquire the lock before `TryStart` has been called), can the validate be updated to match whatever `denyMessage` will be returned?